specs/pvs/corla.pvs

```diff
@@ -781,17 +828,18 @@ END cryptography
 
 %%% \subsection{Randomness}
 
-%%% High quality randomness is key to the legitimacy of any
-%%% risk-limiting audit.
+%%% Choosing a random sample of ballots is key to the legitimacy of any
+%%% risk-limiting audit. Legitimacy depends on a high-quality process for
+%%% the random samples. The Colorado RLA Tool will produce a random order 
+%%% of all the ballots once and for all; then random samples can be created 
```

Comment:
One random *order* of all ballots state-wide allows random samples restricted to counties: for a random sample of size n for County X, simply take the first n ballots from County X that appear in the statewide random order.